url.bs

```diff
@@ -2437,11 +2432,16 @@ background information. [[!HTML]]
 <dl class=switch>
  <dt>"<code>blob</code>"
  <dd>
+  If <a for=/>URL</a>'s <a for=url>blob entry</a> is not null, return <a for=/>URL</a>'s
+  <a for=url>blob entry</a>'s <a for="blob URL entry">environment</a>'s
+  <a for="environment settings object">origin</a>.
```

Comment:
Presumably we'd return `<a for="blob URL entry">origin</a>`? Or do we need to store the environment on this entry as well?

Reply:
The FileAPI spec at least has to store the environment for an entry (to be able to revoke/remove entries when the environment that created them is destroyed). Although of course I could change how the bookkeeping is spec'ed; this just seemed to simplest way from the FileAPI side of things at the time.

I could also change the type/tuple that is returned by resolving a blob URL to something other than what is stored in the blob URL store, but not sure I see benefits to adding that complication (other than separating internal state from what is exposed to other specs).

Reply:
That's fair.